test/e2e/app-dir/app-static/app-static.test.ts

```diff
@@ -2102,10 +2102,10 @@ describe('app-dir static/dynamic handling', () => {
       expect(indexFetchMetrics[0]).toMatchObject({
         url: 'https://next-data-api-endpoint.vercel.app/api/random?page',
         status: 200,
-        cacheStatus: 'skip',
+        cacheStatus: expect.any(String),
```

Comment:
Discussed offline and this is the case because prerendering will now be a cache hit if the worker has seen it before. But this test was really only ensuring that fetch metrics were being written to disk so asserting on specific cache statuses is better suited elsewhere